File: car-brands-api/src/main/java/com/alaasmagi/car_brands_api/domain/FormEntry.java
package com.alaasmagi.car_brands_api.domain;

import lombok.Getter;
import lombok.Setter;

import java.util.List;
import java.util.UUID;

@Getter
@Setter
public class FormEntry {
    private UUID id;
    private String fullName;
    private String contactPhone;
    private boolean validDrivingLicense;
    private List<UUID> selectedCarIds;
}

File: car-brands-client/src/types/form-entry.ts
import type { CarDto, UUID } from './car';

export interface FormEntryRequestDto {
  fullName: string;
  contactPhone: string;
  validDrivingLicense: boolean;
  selectedCarIds: UUID[];
}

export interface FormEntryDto {
  id: UUID;
  fullName: string;
  contactPhone: string;
  validDrivingLicense: boolean;
  selectedCars: CarDto[];
}
